src/InOneWeekend/main.cc

```diff
@@ -75,7 +75,7 @@ void random_spheres(scene& scene_desc) {
 }
 
 
-void main() {
+int main() {
```

Comment:
Commented above, copied here:

Clang6 on my ubuntu was throwing out errors for main that don't return an int. That version of clang is just more sensitive than most compilers. It's a small thing, but it was an error for me, I assumed it would be an error for a small slice of the audience

I don't care either way.

Reply:
I'll let you decide. I mostly don't like the anachronism of int-returning but never actually returning, as a common compiler backwards-compatible sleight-of-hand.

You choose:
1. `void main()`, no explicit return.
2. `int main()`, return 0.

Either works for me, and from your input, `int main()` may cause fewer warnings for readers of all platforms.